### ISSUE TITLE
SQLite's real(float) datatype is not supported

### ISSUE THREAD

**Issue by ccwang002** [duckdblabs/sqlite_scanner#12]
I got an invalid error `Invalid Error: FLOAT` while trying to read an attached SQLite table with a real/float column. Here are the steps to reproduce:

Create a SQLite database:
```bash
echo "CREATE TABLE my (a REAL); INSERT INTO my VALUES (10.34), (0.042);" | sqlite3 my.db
```

Attach it from DuckDB (`./build/release/duckdb -unsigned`):
```text
D LOAD 'build/release/extension/sqlite_scanner/sqlite_scanner.duckdb_extension';
D CALL sqlite_attach('my.db');
D SELECT * FROM my;
Error: Invalid Error: FLOAT
```

I was able to load tables of other data types (e.g., text, integer).

Also, thank you for making this extension. Now it's much easier to transfer/load data from existing SQLite databases to DuckDB.

**Comment by aley1** [comment]
I've encountered the same error, hopefully this will get fixed soon.

**Comment by Mytherin** [comment]
Thanks for the report! This should be fixed by #18 now.
